### DISCUSSION TITLE
What do I have to do to get this?

### DISCUSSION THREAD

**Discussion by EeeasyCode** [Schweinepriester/github-profile-achievements#273] [Q&A] [answered]
Hello, I am a student majoring in computer engineering at a university in Korea. 
What should I do to get the Galaxy Brain?
I believe you will understand my poor English skills.
Thank you.

**Comment by thenithinbalaji** [comment, marked as answer]
# How do you get Galaxy brain?
+ Answer at least 2 discussions. 
+ Take note, you cannot mark your own answer as the answer to your own question.
+ Answer questions asked by users in discussions and get marked. 
+ For example, if you mark this answer as "Mark as Answer", then I'll get a point for Galaxy Brain. 
+ Similarly, you will get a point when someone marks your answer as the answer in some discussion. 

![image](https://user-images.githubusercontent.com/73932121/179235281-69524f35-21a5-4c53-a89d-35dccc87ebf0.png)

# This badge not only has the base version, but also tiers.

![image](https://user-images.githubusercontent.com/73932121/179235882-033cffe0-4bbd-44e2-a3be-b1a078745b05.png)

**Reply by EeeasyCode** [reply]
thank you so much!!!!
